apis/v1alpha2/policy_types.go

```diff
@@ -120,3 +122,81 @@ const (
 	// policy is attached to an invalid target resource.
 	PolicyReasonTargetNotFound PolicyConditionReason = "TargetNotFound"
 )
+
+// PolicyAncestorStatus describes the status of a route with respect to an
+// associated Ancestor.
+//
+// Ancestors refer to objects that are either the Target of a policy or above it in terms
+// of object hierarchy. For example, if a policy targets a Service, an Ancestor could be
+// a Route or a Gateway.
```

Comment:
This seems to disagree with https://github.com/kubernetes-sigs/gateway-api/pull/2448/files#diff-4333085192c93205580c7dd9213f10fe061e77245adfa23310487ae0924c7dbaR144-R145, which claims the Gateway as the ancestor object.  Could you explain under what circumstance a Route would be an ancestor of a Service, but not the Gateway?

Reply:
A Route is always _an_ ancestor of the Service, but it is probably not _the relevant_ ancestor of the Service unless the Route does not attach to a Gateway, because all Routes that have Services with BackendTLSPolicies attached, that themselves attach to a Gateway that supports BackendTLSPolicy will have the same behaviors, in general.

Of course, this is only a _recommendation_, not a _specification_, so we can't _make_ people do anything.

I'll try to clarify this wording a bit.